frontend/src/components/project/ProjectInfo.vue

```diff
@@ -29,16 +29,25 @@
                 </v-card-title>
                 <div v-html="renderQuillContent(project.description)"></div>
             </v-card-item>
+            <v-btn color="blue" v-if="isTeacher" :to="`/project/${project.id}/submissions`">
+                {{ $t("project.submissions_list_teacher") }}
+            </v-btn>
```

Comment:
```suggestion

            <v-card-actions>
                <v-btn v-if="isTeacher" :to="`/project/${project.id}/submissions`" class="...">
                    {{ $t("project.submissions_list_teacher") }}
                </v-btn>
            </v-card-actions>
```
- `v-card-actions` voor de padding
- styling met css